components/store/StoreHours.js

```diff
@@ -0,0 +1,78 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { View } from 'react-native';
+import Colors from '../../constants/Colors';
+import { constructHoursDict } from '../../lib/mapUtils';
+import { Body, TabSelected } from '../BaseComponents';
+
+/**
+ * @prop
+ * */
+
+export default function StoreHours({ hours }) {
+  // e.g. hours = "8:30am-8:30pm Mon-Sat"
+
+  const daysOfTheWeek = {
+    Sun: 'Sunday',
+    Mon: 'Monday',
+    Tu: 'Tuesday',
+    Wed: 'Wednesday',
+    Thurs: 'Thursday',
+    Fri: 'Friday',
+    Sat: 'Saturday',
+  };
+
+  const daysOfTheWeekFull = Object.values(daysOfTheWeek);
+
+  const today = new Date();
+  const todaysDayIndex = today.getDay();
+  const todaysDay = daysOfTheWeekFull[todaysDayIndex];
+
+  const hoursList = dictHours => {
+    return (
+      <View style={{ display: 'flex', flexDirection: 'row' }}>
+        {/* Days of the week */}
+        <View style={{ width: '50%' }}>
```

Comment:
Nit: Maybe adjust this 50-50 ratio to more 40-60 just to make sure we don't get weird wrapping/truncating edge cases on narrower screens.